components/reports/beetles/ReportBeetleRiskMaps.vue

```diff
@@ -0,0 +1,170 @@
+          name="beetle_risk_maps_model_selection"
+          native-value="4"
+          >MRI CGCM3</b-radio
+        >
+      </b-field>
+    </div>
+    <div class="is-size-6 mb-4">
+      <b-field label="Snowpack Level">
+        <b-radio
+          v-model="beetle_risk_maps_snowpack_selection"
+          name="beetle_risk_maps_snowpack_selection"
+          native-value="0"
+          >Low</b-radio
+        >
+        <b-radio
+          v-model="beetle_risk_maps_snowpack_selection"
+          name="beetle_risk_maps_snowpack_selection"
+          native-value="1"
+          >Medium</b-radio
+        >
+      </b-field>
+    </div>
+    <div class="columns is-flex-direction-row is-centered">
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          model="0"
+          scenario="0"
+          era="0"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          :model="beetle_risk_maps_model_selection"
+          scenario="1"
+          era="1"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          :model="beetle_risk_maps_model_selection"
+          scenario="1"
+          era="2"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          :model="beetle_risk_maps_model_selection"
+          scenario="1"
+          era="3"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+    </div>
+    <div class="columns is-flex-direction-row is-centered">
+      <div class="minimap-container my-4 p-1"></div>
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          :model="beetle_risk_maps_model_selection"
+          scenario="2"
+          era="1"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          :model="beetle_risk_maps_model_selection"
+          scenario="2"
+          era="2"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+      <div class="minimap-container my-4 p-1">
+        <ReportBeetleRiskMap
+          :model="beetle_risk_maps_model_selection"
+          scenario="2"
+          era="3"
+          :snowpack="beetle_risk_maps_snowpack_selection"
+        />
+      </div>
+    </div>
+    <div class="legend">
+      <div
+        class="color is-flex is-flex-direction-row"
+        v-for="(beetleRisk, key, index) in beetleRisks"
+      >
+        <div
+          class="swatch"
+          :style="'background-color: ' + beetleRisk['color']"
+        />
+        <div>{{ beetleRisk['label'] }}</div>
+      </div>
+    </div>
```

Comment:
The legend items look a bit off with the "High" category on a row of its own here:

![image](https://user-images.githubusercontent.com/2566292/191819352-36e83572-cf5e-44e6-96b9-7f81b93c9291.png)

However, I think if we remove the whole mini-map section from the fixed-width container (described in my comment above), this may fix itself by giving the legend more room to breathe, and hopefully list all the legend items in one row.